app/components/visualizer/main-viz/templates/Chart.jsx

```diff
@@ -0,0 +1,42 @@
+import DynamicPieChart from "./DynamicPieChart.js";
+
+class Chart extends React.Component {
+  componentDidMount() {
+    let el = React.findDOMNode(this);
```

Comment:
Yeah its really nice and new as of es6. It gives you block scoping in JS. let is the new var! ha.